From 70e52bdb507276320a35249c87161dd451ece1c8 Mon Sep 17 00:00:00 2001
From: Brian Lamar <brian.lamar@rackspace.com>
Date: Thu, 29 Sep 2011 15:55:42 -0400
Subject: [PATCH] Added ==1.0.4 version specifier to kombu in pip-requires to
 ensure tests pass in a clean venv.

Fixes bug 862629.

(cherry picked from commit eb4bd86f65a440b39804d16e477aeb77f31e11c1)

Change-Id: I7f8f327633eed21067cfb48bade84da04bdf1780
---
 tools/pip-requires |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/pip-requires b/tools/pip-requires
index 232887c..26c639b 100644
--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -8,7 +8,7 @@ anyjson==0.2.4
 boto==1.9b
 carrot==0.10.5
 eventlet
-kombu
+kombu==1.0.4
 lockfile==0.8
 lxml==2.3
 python-novaclient==2.6.0
-- 
1.7.6.5

